src/FileSystem-Core/AbstractFileReference.class.st

```diff
@@ -530,6 +530,34 @@ AbstractFileReference >> humanReadableSize [
 	^ self size humanReadableSIByteSize
 ]
 
+{ #category : #delegated }
+AbstractFileReference >> ifAbsent: aBlock [
+	"Return self, or evaluate the block if I do not exist"
+
+	^ self exists ifTrue: [ self ] ifFalse: [ aBlock cull: self ]
+]
+
+{ #category : #delegated }
+AbstractFileReference >> ifAbsent: anotherBlock ifExists: aBlock [
```

Comment:
I would probably do not add another combination of words, but ok

Reply:
It made sense when we were copying the pattern for "ifTrue:ifFalse", "ifNil:ifNotNil:", "ifEmpty:ifNotEmpty:" but I agree this one seems weird. I'll remove it!